cdap-app-templates/cdap-etl/cdap-etl-batch/src/main/java/co/cask/cdap/etl/batch/spark/SparkTransformExecutorFactory.java

```diff
@@ -51,18 +52,19 @@ public void transform(Object input, Emitter emitter) throws Exception {
   private final PluginContext pluginContext;
   private final long logicalStartTime;
   private final Map<String, String> runtimeArgs;
-  private final boolean isPreGroup;
+  // whether this is before the group step of an aggregator, or before a spark compute stage
+  private final boolean isBeforeBreak;
```

Comment:
yeah i wasn't very happy with this name either.  I may just see if I can refactor the spark compute code path